src/cpu/o3/lsq_unit.hh

```diff
@@ -289,6 +289,8 @@ class LSQUnit
     using LoadQueue = CircularQueue<LQEntry>;
     using StoreQueue = CircularQueue<SQEntry>;
 
+    std::vector<LSQRequest*> inflightLoads;
```

Comment:
After [#265](https://github.com/OpenXiangShan/GEM5/pull/265) , once an load instruction is replayed due to a cache miss, the request at the corresponding position in the LQ will be set to as `nullptr`, and this miss request will not be tracked. Using `request` to determine if an instruction is inflight might lead to errors.

Maybe use `seqNum` to track inflightLoads can solve these errors